homework/homework_part_2/js/app.js

```diff
@@ -0,0 +1,75 @@
+// 1. The difference between interpolation and concatenation is the syntax.
+// 	  Interpolation looks like this: console.log(`This is how ${interpolation} works')
+// 	  Concatenation looks like this: console.log('We ' + 'can ' + concatenate ' like this!')
+
+// 2. DRY stands for Don't Repeat Yourself. You should pay attention to it because
+// 	  it directly effects your code and how efficiently you write it. We can use
+// 	  while and for loops to make sure that we are writing DRY code.
+
+// 3. Declaring a variable tells the compiler that there is a variable. Defining it
+// 	  gives the variable a value.
+
+// 4. Always use const instead of let unless the system yells at you to do otherwise
```

Comment:
yesssssssssss and by "yells at you" you mean Uncaught TypeError: Assignment to constant variable.